templates/admin/login/login.html.twig

```diff
@@ -38,15 +38,11 @@
         <a href="https://www.microsoft.com/edge" target="_blank" rel="noopener noreferrer" title="Edge"><img src="/bundles/pimcoreadmin/img/login/edge.svg" alt="Edge"/></a>
     </div>
 
-    <a href="#" onclick="showLogin();">{{ 'Click here to proceed'|trans([], 'admin') }}</a>
+    <a id="proceed_with_unsupported_browser" href="#" >{{ 'Click here to proceed'|trans([], 'admin') }}</a>
 </div>
 
-<script type="text/javascript">
-    function showLogin() {
-        document.getElementById('loginform').style.display = 'block';
-        document.getElementById('browserinfo').style.display = 'none';
-    }
-</script>
+<script src="/bundles/pimcoreadmin/js/pimcore/browserNotSupported.js"></script>
```

Comment:
LGTM and works fine. 

Just one input: i'd suggest to delete `browserNotSupported` and move some line below

https://github.com/pimcore/admin-ui-classic-bundle/blob/f97c7f5475c4504cbe1b3793b8ee2af9c600e860/templates/admin/login/login.html.twig#L68

```
{% if not browserSupported %}
document.getElementById('proceed_with_unsupported_browser').addEventListener('click', showLogin);

/**
 * @private
 */
function showLogin() {
    document.getElementById('loginform').style.display = 'block';
    document.getElementById('browserinfo').style.display = 'none';
}
{% endif %}
```

reason being this code is used only here and only when the button `a#proceed_with_unsupported_browser` is present, although won't hurt, but doesn't need to be in an external js as it's never reusable.

Reply:
Alternatively should move `<script src="/bundles/pimcoreadmin/js/pimcore/browserNotSupported.js"></script>` within `{% if not browserSupported %}` block but not the best practice in terms that the js is neither in header or footer but in middle of the code

Reply:
Maybe use https://pimcore.com/docs/platform/Pimcore/MVC/Template/Template_Extensions/HeadScript ?